src/components/Reports/GDPR/SubscriberGdpr.js

```diff
@@ -60,11 +61,25 @@ const SubscriberGdpr = ({ location, dependencies: { dopplerApiClient } }) => {
           return field;
         });
 
+        const tabsProperties = [
+          {
+            url: `/reports/subscriber-history?email=${subscriber.email}`,
+            descriptionId: 'subscriber_history.title',
+            active: false,
+          },
+          {
+            url: `/reports/subscriber-gdpr?email=${subscriber.email}`,
+            descriptionId: 'subscriber_gdpr.page_title',
+            active: true,
+          },
+        ];
+
         setState({
           loading: false,
           subscriber: subscriber,
           fields: fields,
           email: email,
+          tabsProperties: tabsProperties,
```

Comment:
its neccesary put in the state? maybe never change

Reply:
Good catch @fcoronelmakingsense!